java/dbschema/nbproject/org-netbeans-modules-dbschema.sig

```diff
@@ -2911,12 +2911,13 @@ meth public static void setDescriptionAsResource(org.openide.loaders.DataObject,
 meth public static void setIterator(org.openide.loaders.DataObject,org.openide.loaders.TemplateWizard$Iterator) throws java.io.IOException
  anno 0 java.lang.Deprecated()
 meth public void setTargetFolder(org.openide.loaders.DataFolder)
+meth public void setTargetFolderLazy(java.util.function.Supplier<org.openide.loaders.DataFolder>)
```

Comment:
I do remember this API change. Done by @errael, I think.

Reply:
PR #1991